spec/helpers/application_helper_spec.rb

```diff
@@ -48,7 +49,8 @@
 
   describe '#twitter_url' do
     it 'returns twitter url for given non nil handle' do
-      helper.twitter_url('CodeMontage').should eq('http://twitter.com/CodeMontage')
+      helper.twitter_url('CodeMontage')
+        .should eq('http://twitter.com/CodeMontage')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.